### ISSUE TITLE
🦠 Model Request: Deep Learning: Drug-Target Interaction Prediction

### ISSUE THREAD

**Issue by AdedejiAdewole** [ersilia-os/ersilia#648]
### Model Name

DTI Prediction Model

### Model Description
Python generated DL framework that takes SMILES string and protein amino acid pairs as input. They are fed into molecular encoders to convert compounds and proteins to a vector representation.
These embedded  representations are fed into the decoder to generate prediction outputs, both continuous binding scores and binary outputs indicating if a protein binds to a compound. A detection happens if task is a regression or classification and uses the correct loss function and evaluation method. 
1. MSE for regression 
2. Binary Cross Entropy for classification

### Slug

drug-target-prediction

### Tag

Target identification, Embedding

### Publication

https://doi.org/10.1093/bioinformatics/btaa1005

### Source Code

https://github.com/kexinhuang12345/DeepPurpose

### License

BSD 3-Clause License
